Makefile

```diff
@@ -41,12 +41,20 @@ test: generate manifests
 
 # Build manager binary
 manager: generate fmt vet
```

Comment:
We already have linting enabled, you can use the target `golint` that is containing all the required rules.